### ISSUE TITLE
Zigbee Energy Profile Devices

### ISSUE THREAD

**Issue by monkeyatcomputer** [home-assistant/home-assistant.github.io#31669]
### Feedback

The article states that there aren't any devices with local API. I don't believe this to be correct.

I'm using a Rainforest Automation EAGLE-200 with Rainforest Eagle integration and it's reading instantaneous usage (W), and import/export counters (kWh) via the eagle's local API. Believe the current version is called an "EAGLE 3".

I'm in the state of Victoria, Australia where the state government mandated the use of smart meters including Zigbee radios (meant for in home energy displays, and demand response devices). Our electricticity distributor AusNet has a website where we can register the deivce and a few days later it just starts working.

### URL

https://www.home-assistant.io/docs/energy/electricity-grid/

### Version

2024.2.5

**Comment by agoode** [comment]
Yes, this integration uses a local API: https://www.home-assistant.io/integrations/rainforest_eagle/

**Comment by Hedda** [comment]
See pull request -> https://github.com/home-assistant/home-assistant.io/pull/33461

But note that the Rainforest Eagle integration does not support have native support for Zigbee Smart Energy Standard (devices that uses Zigbee Smart Energy Profile), as instead they have a propriatory gateway can translate and covert it to their own custom API, so the integration component itself inside Home Assistant does not natively support the Zigbee Smart Energy Standard.

Also see -> https://www.home-assistant.io/integrations/zha#limitations

So it is really like comparing the [native ZHA integration](https://www.home-assistant.io/integrations/zha) with other integrations that communicate only with for third-party gateways/bridges/hubs using the manufacturers custom APIs, such as Philips Hue Bridge, IKEA Tradfri Gateway, and Samsung SmartThings Hub. There is no Zigbee commands in the integration component itself inside Home Assistant for any of those as they totally abstract away anything Zigbee specific so they devices themself could just as well be using any other IoT protcol and niether the developers or end-users would know the difference inside the integration.

#### **_Connect to devices that uses Zigbee Smart Energy Standard_**

**_[Zigbee Smart Energy Standard](https://csa-iot.org/all-solutions/smart-energy/) is a wireless protocol designed by the CSA (Connectivity Standards Alliance, formerly the Zigbee Alliance) to specifically target industrial and commercial utilities to enable real-time monitoring of energy consumption and actively managing energy consumption at the end-user level. This wireless protocol is supported by some utility meters for monitoring electricity, gas, and water consumption in the US, EU, UK, and Australia._**

**_It is also [referred to “Zigbee SE” or "ZSE" profile in the specification](https://zigbeealliance.org/wp-content/uploads/2019/11/docs-07-5356-19-0zse-zigbee-smart-energy-profile-specification.pdf), and note that it should not be confused with other Zigbee profiles, because Zigbee Smart Energy Profile is not part of the common "Zigbee 3.0" suite of profiles that are supported tHome Assistant's native Zigbee Home Automation integration, (as the Home Assistant's ZHA integration currently only have implemented support for other Zigbee profiles that are specifically made for home automation, lighting or occupancy, and not devices that just use the Zigbee Smart Energy" Profile)._**

**_Currently, the only solution we are aware of that can use devices that only support the "Zigbee Smart Energy" profile is Rainforest Automation's EAGLE gateway appliances that can be used via the Rainforest Eagle integration for Home Assistant, (those third-party gateways made for reading Australian electricity meters, though it abstracts and translates that protocol to present the data via their custom local API, and is as such not a native Zigbee Smart Energy" integration)._**

**Comment by HSB-collab** [comment]
> ### Feedback
> The article states that there aren't any devices with local API. I don't believe this to be correct.
> 
> I'm using a Rainforest Automation EAGLE-200 with Rainforest Eagle integration and it's reading instantaneous usage (W), and import/export counters (kWh) via the eagle's local API. Believe the current version is called an "EAGLE 3".
> 
> I'm in the state of Victoria, Australia where the state government mandated the use of smart meters including Zigbee radios (meant for in home energy displays, and demand response devices). Our electricticity distributor AusNet has a website where we can register the deivce and a few days later it just starts working.

Are you able to tell me where you got the Eagle-200. I am also in Victoria but I am struggling to find an Eagle-200 online

**Comment by ravdanin** [comment]
> > ### Feedback
> Are you able to tell me where you got the Eagle-200. I am also in Victoria but I am struggling to find an Eagle-200 online
Eagle-200 is discontinued and replaced with Eagle 3. The only place I have seen it brand new for sale is on the Rainforest Automation website: https://rainforestautomation.com/us-retail-store/

**Comment by HSB-collab** [comment]
> > > ### Feedback
> > 
> > 
> > Are you able to tell me where you got the Eagle-200. I am also in Victoria but I am struggling to find an Eagle-200 online
> > Eagle-200 is discontinued and replaced with Eagle 3. The only place I have seen it brand new for sale is on the Rainforest Automation website: https://rainforestautomation.com/us-retail-store/

I bought the device off eBay but unfortunately Rainforest Automation would not support it because they told "it had gone missing from a DHL warehouse in 2021". You can't use the Eagle 200 unless it is authorised via the Rainforest cloud so I got my money back

**Comment by ravdanin** [comment]
My understanding is if it's already paired it does not need a cloud connection to work. Without cloud connection there is no firmware updates - but frankly all they offer on their cloud looks like a summer project of some intern. The pairing itself is another question. Not sure it requires cloud connection either - could be just some magic on a side of power utility company. In my case it worked out in a few hours after few online attempts to register it with power utility.

**Comment by HSB-collab** [comment]
> My understanding is if it's already paired it does not need a cloud connection to work. Without cloud connection there is no firmware updates - but frankly all they offer on their cloud looks like a summer project of some intern. The pairing itself is another question. Not sure it requires cloud connection either - could be just some magic on a side of power utility company. In my case it worked out in a few hours after few online attempts to register it with power utility.

I don't think my Eagle had ever been paired and my understanding is that in order to validate the device's authority to read the meter, the utility does it via the Rainforest cloud. I'd be happy for someone to tell me I am wrong!
